packages/cli/src/databases/entities/User.ts

```diff
@@ -62,7 +62,7 @@ export class User {
 	@PrimaryGeneratedColumn('uuid')
 	id: string;
 
-	@Column({ length: 254 })
+	@Column({ length: 254, nullable: true })
```

Comment:
Attempting to type this as `string | null` at L68 triggers [this issue](https://github.com/typeorm/typeorm/issues/1358).